common/doc/swdesign/README.md

```diff
@@ -332,6 +332,45 @@ Needs:
 - impl
 - utest
 
+#### Workload naming convention
+`swdd~common-workload-naming-convention~1`
```

Comment:
We have not traced this and the next requirement in code and in the tests yet.

Reply:
Done